specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/DeletedAccountGet.json

```diff
@@ -9,16 +9,17 @@
   "responses": {
     "200": {
       "body": {
-        "id": "/subscriptions/{subscription-id}/providers/Microsoft.Storage/locations/eastus/deletedAccounts/sto1125",
-        "name": "sto1125",
-        "type": "Microsoft.Storage/deletedAccounts",
-        "properties": {
-          "storageAccountResourceId": "/subscriptions/{subscription-id}/resourceGroups/sto/providers/Microsoft.Storage/storageAccounts/sto1125",
-          "location": "eastus",
-          "restoreReference": "sto1125|2020-08-17T03:35:37.4588848Z",
-          "creationTime": "2020-08-17T03:35:37.4588848Z",
-          "deletionTime": "2020-08-17T04:41:37.3442475Z"
-        }
+        "value": [
+          {
+            "key": "sto1125|2020-08-17T03:35:37.4588848Z",
```

Comment:
The example is not aligned with following swagger, this might be why the swagger validation fail in CI.
Per [swagger defination](https://github.com/Azure/azure-rest-api-specs/blob/b58afa291e0e40449661096b6005140ce9b8d451/specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json#L3050), there will be a `"properties": {}` tag in each list/get out account, but not such tag in example. Please make sure swagger and example are aligned.